docker-compose/docker-compose.yaml

```diff
@@ -209,6 +212,10 @@ services:
       - PM4ML_ENABLED=${PM4ML_ENABLED:-false}
       - ENABLE_BACKEND_EVENT_HANDLER=false
       - ENABLE_FSPIOP_EVENT_HANDLER=false
+      - FSPIOP_API_SERVER_MAX_REQUEST_BYTES=209715200
+      - BACKEND_API_SERVER_MAX_REQUEST_BYTES=209715200
+      - SUPPORTED_CURRENCIES=''
```

Comment:
You may need this field to be set when you are working on payee side implementation.